### PR TITLE
Minor Club remap: added windoor in front of booze dispenser, replaced…

### PR DESCRIPTION
**Pull request by maikilangiolo**
## About The Pull Request

Minor Club remap: added windoor in front of booze dispenser, replaced three walls with windows, added 'take-out' table to kitchen.

Condi-master and sink have been switched, since the sink looked ugly on the window and this way it is also closer to the cooking area. 

(p.s. I got no clue how to make pull requests and operate git, so this PR might be fucked.)

## Why It's Good For The Game

Well, it allows people to see inside the kitchen, and even come pick up food if they order before hand. Not only does it allow customers to see the chef cooking (and the chef can see customers that are waiting for him to come out), and adds the possibility to chat through the windows, but it also (hopefully) should reduce the instances of theft because people don't see anyone manning the club counter and climb it to steal drinks or food by breaking into the kitchen.

It does not make breaking into club any easier, since it would be faster to bolt open the door or smash it with a crowbar, or even slide under the flaps, rather than break the new windows/windoors and clib over the table or low wall.

## Changelog
:cl:

tweak: changed three walls in the kitchen to windows
tweak: switched condi-master and sink positions
tweak: moved the air alarm to above the drink showcase
add: added a new double windoor set-up for take-out
add: added windoor in front of booze dispenser to reduce or delay theft

/:cl:

### CODE REVIEW

**Comment by maikilangiolo** [comment]
forgot to link the image in the PR: https://ibb.co/9380DRP

**Comment by fernandos13** [comment]
so one problem in the file. cev eris map. theres some lines with _**step_x = -2**_   you can just text eddit with VS or even notepad++ and remove it. literally delete this lines"

just the step_x= number

part of it. and any     ,    or  ; that follows. it will solve the compiling error

File: maps/CEVEris/_CEV_Eris.dmm [hunk 1]
```diff
@@ -25129,7 +25129,8 @@
 /area/eris/maintenance/section2deck4central)
 "beo" = (
 /obj/machinery/light{
-	dir = 1
+	dir = 1;
+	step_x = -2
```
Comment (fernandos13) [inline maps/CEVEris/_CEV_Eris.dmm:25133]: like here. just delete the line with step_x = -2

Comment (fernandos13) [reply maps/CEVEris/_CEV_Eris.dmm:25133]: and every other line thats like this one

**Comment by fernandos13** [comment]
i made the changes in a file 
heres the file clean of ### step_x= _something_ and step_x= _something_
[_CEV_Eris.zip](https://github.com/discordia-space/CEV-Eris/files/5225558/_CEV_Eris.zip)

**Comment by fernandos13** [comment]
i made a PR to your branch. with its changes

File: maps/CEVEris/_CEV_Eris.dmm (hunk 1 above)
Comment (fernandos13) [reply maps/CEVEris/_CEV_Eris.dmm:25133]: solved

**Comment by Firefox13** [comment]
(merge conflict)
What I do suggest, is to add shutters to the windows that start closed. And a button in kitchen to control them.

**Review by genessee596 — APPROVED** [review]
Delegate stamp of approval.

The takeout window makes it easier for people to exchange food at the kitchen counter without having to wade through the _numerous_ people (read: almost none) that are always sitting at the bar. A good QoL change.

Needs maintainer review before being mergable.

**Review by Iglpigl — CHANGES_REQUESTED** [review]
Resolve conflicts, looks good otherwise.

**Comment by maikilangiolo** [comment]
fernando should have solved my own clumsiness and the conflicts that spawned when I mapped? I honeslty got no clue what's wrong, I just mapped some shit in and this happened.

**Comment by Firefox13** [comment]
> fernando should have solved my own clumsiness and the conflicts that spawned when I mapped? I honeslty got no clue what's wrong, I just mapped some shit in and this happened.

Mergeconflict happens when someone's PR edits the file your PR edits and it gets merged before yours.

**Comment by maikilangiolo** [comment]
so, can I solve it or I can happily leave the task to someone that knows what he's fucking doing?

**Comment by Firefox13** [comment]
> so, can I solve it or I can happily leave the task to someone that knows what he's fucking doing?

yes, you need to save your changes(outside of the repo), rebase(update to master, it will wipe your chanhes) and re-apply them again

**Comment by maikilangiolo** [comment]
so, how do I save the changes then? copy pasting the map file somewhere else will just use the old map before the rebase when I overwrite it.

**Comment by Firefox13** [comment]
> so, how do I save the changes then? copy pasting the map file somewhere else will just use the old map before the rebase when I overwrite it.

You copypaste the map outside of the repo
you update
you shove it back, but do not replace the map file - you manually ctrl-c ctrl-v changes into actual repo map that was updated

**Comment by maikilangiolo** [comment]
i fucked up my git everything, fork included. so I will just close this PR, nuke my fork, eat a bullet and start again.

**Pull request closed without merging**